system/user/addons/wb_category_select/ft.wb_category_select.php

```diff
@@ -596,15 +624,33 @@ private function _add_js_css($field_id, $cell)
 			ee()->session->set_cache('wb_category_select', 'cp_assets_set', true);
 		}
 
-		if ($cell)
+		if ($cell == 'matrix')
 		{
-			ee()->cp->add_to_foot('
-			<script type="text/javascript">
-				(function($) {
-					Matrix.bind("wb_category_select", "display", function(cell){ $(this).find("select").multi(); });	
-				})(jQuery);
-			</script>
-			');
+			if ( ! ee()->session->cache('wb_category_select', 'cp_matrix_set'))
```

Comment:
You should combine this conditional with the conditional on line 627:

```php
if ($cell == 'matrix' && ! ee()->session->cache('wb_category_select', 'cp_matrix_set'))
```

Reply:
I think this may cause an issue  where if we combined the conditionals we would end up with the last condition being met unnecessarily where the previous conditions are false when cached.

Reply:
Oh, good point. Ignore these then.